pkg/ast/pipesearch/searchQueryParser.go

```diff
@@ -473,6 +473,11 @@ func parseColumnsCmd(node *structs.OutputTransforms, qid uint64) (*QueryAggregat
 	}
 
 	aggNode.OutputTransforms.MaxRows = node.MaxRows
+	aggNode.OutputTransforms.Tail = node.Tail
+	aggNode.OutputTransforms.TailRows = node.TailRows
+	if node.TailRequest != nil {
+		aggNode.OutputTransforms.TailRequest = node.TailRequest
+	}
```

Comment:
Why do you need this `if` for `TailRequest` and not need it for `Tail` and `TailRows`?

Reply:
addressed in the latest commit